apps/backend/src/modules/auth/auth.controller.ts

```diff
@@ -98,6 +137,47 @@ class AuthController extends BaseController {
 		});
 	}
 
+	/**
+	 * @swagger
+	 * /auth/forgot-password:
+	 *    post:
+	 *      tags:
+	 *        - Authentication
+	 *      description: Sends link for update password on specified email
+	 *      requestBody:
+	 *        required: true
+	 *        content:
+	 *          application/json:
+	 *            schema:
+	 *              type: object
+	 *              properties:
+	 *                email:
+	 *                  type: string
+	 *                  format: email
+	 *      responses:
+	 *        200:
+	 *          description: Successful operation
+	 *          content:
+	 *            application/json:
+	 *              schema:
+	 *                type: object
+	 *                properties:
+	 *                  success:
+	 *                    type: boolean
+	 */
+	private async forgotPassword({
+		body: { email },
+	}: APIHandlerOptions<{
+		body: AuthForgotPasswordRequestDto;
+	}>): Promise<APIHandlerResponse> {
+		const success = await this.authService.forgotPassword(email);
+
+		return {
+			payload: { success },
+			status: HTTPCode.OK,
```

Comment:
```suggestion
			status: HTTPCode.CREATED,
```
Post should return status 201